### PR TITLE
Fix MangaKakalot

### PR DESCRIPTION
**Pull request by plskz**
Proof:

# **mangakakalot**

image

image

image

image

# manganato
image

image

# readmanganato
image

image

### CODE REVIEW

File: websites/M/MangaKakalot/presence.ts [hunk 1]
```diff
@@ -1,102 +1,114 @@
...
+          ".story-info-right > h1"
+        ).textContent;
+      }
+
+      break;
     }
   }
 
-  if (presenceData.details == null) {
+  if (presenceData.details === null) {
```
Comment (SlowLife1661) [inline websites/M/MangaKakalot/presence.ts:110]:
```suggestion
  if (!presenceData.details) {
```

File: websites/M/MangaKakalot/presence.ts [hunk 2]
```diff
@@ -1,102 +1,114 @@
 const presence = new Presence({
-  clientId: "698217762660548799"
-});
-
-const browsingStamp = Math.floor(Date.now() / 1000);
+    clientId: "698217762660548799"
+  }),
+  browsingStamp: number = ~~(Date.now() / 1000);
```
Comment (SlowLife1661) [inline websites/M/MangaKakalot/presence.ts:4]: No need
```suggestion
  browsingStamp = ~~(Date.now() / 1000);
```

**Review by Slowlife01 — APPROVED** [review]

**Comment by ItsTonyBTW** [comment]
Hello @Bas950, my Manganato doesnt seem to work correctly. It shows presence in the home page and genre, but when i select a manga or open a chapter, it immediately stops showing. Please help.
Also i go to manga kakalot, select a manga there, but it opens up in manganato. Is there a fix to this issuse? Anticipating a positive reponse, Tony.
image_2021-07-02_140107
look here it works fine but when i open manga it dissappears

**Comment by ririxidev** [comment]
@ItsTonyBTW it's not a place for this kind of stuff. If you don't see you just wrote on pull request that is going to fix this stuff.

**Comment by plskz** [comment]
@ririxidev should I add myself as a contributors? XD

**Review by rxri — APPROVED** [review]

**Comment by ririxidev** [comment]
meh

**Pull request merged**